roles/download/defaults/main.yml

```diff
@@ -493,9 +493,10 @@ haproxy_image_tag: 2.3
 
 # Coredns version should be supported by corefile-migration (or at least work with)
 # bundle with kubeadm; if not 'basic' upgrade can sometimes fail
-coredns_image_is_namespaced: "{{ (kube_major_version | regex_replace('^v', '') | float) >= 1.21 }}"
 
 coredns_version: "v1.8.0"
+coredns_image_is_namespaced: "{{ (kube_version is version('v1.21.0','>=')) or (coredns_version is version('v1.7.1','>=')) }}"
```

Comment:
The linked discussion shows the subpath depends on coredns_version(v1.7.1+).
Why here needs to depend on kube_version also?

Reply:
ah, that is for the existing commit a0ee5690919ad72725ca3bffe2dc00dff9c8b3d0
It seems fine to keep here.

Reply:
/lgtm